// containers/basket/formDelivery.js
import  React, { Component } from 'react';

import {connect} from 'react-redux'
import {startAddOrder} from 'actions'

class FromDelivery extends Component {
    constructor(props) {
        super(props);
        this.btnSendInfo = this.btnSendInfo.bind(this);

    }
    componentWillMount(){
        this.setState({
            address : '',
            name : '',
            phone : '',
            email : ''
        });
    }
    handleEmailChange (e) {
        this.setState({email: e.target.value});
    }
    handlePhoneChange (e) {
        this.setState({phone: e.target.value});
    }
    handleAddressChange (e) {
        this.setState({address: e.target.value});
    }
    handleNameChange (e) {
        this.setState({name: e.target.value});
    }
    handleMessageChange (e) {
        this.setState({message: e.target.value});
    }

    btnSendInfo(event)
    {
        event.preventDefault();
        startAddOrder(this.props.order, this.state.name, this.state.email, this.state.phone, this.state.address, this.props.price, this.state.message)
    }

    onChangeText()
    {
        //event.preventDefault();
    }
    render(){
        return (
            <div className="jumbotron-sm">
                <div className="container">
                    <div className="row">
                        <div className="col-md-8">
                            <div className="well-sm">
                                <form>
                                    <div className="row">
                                        <div className="col-md-6">
                                            <div className="form-group">
                                                <label for="name">
                                                    Имя</label>
                                                <input type="text" className="form-control" id="name" placeholder="Розарио" value={this.state.name} required="required" onChange={(e)=> this.handleNameChange(e)}/>
                                            </div>

                                            <div className="form-group">
                                                <label for="name">
                                                    Адрес доставки</label>
                                                <input type="text" className="form-control" id="adress" placeholder="улица, квартира" value={this.state.address} required="required" onChange={(e)=> this.handleAddressChange(e)}/>
                                            </div>

                                            <div className="form-group">
                                                <label for="email">
                                                    Email</label>
                                                <div className="input-group">
                                            <span className="input-group-addon"><span className="glyphicon glyphicon-envelope"></span>
                                            </span>
                                                    <input type="email" className="form-control" id="email" value={this.state.email} placeholder="your@email.com" required="required" onChange={(e)=> this.handleEmailChange(e)}/></div>
                                            </div>
                                            <div className="form-group">
                                                <label for="email">
                                                    Телефон</label>
                                                <input type="text" id="phone" className="form-control" value={this.state.phone} placeholder="+79092345678" required="required" onChange={(e)=> this.handlePhoneChange(e)}/></div>

                                        </div>
                                        <div className="col-md-6">
                                            <div className="form-group">
                                                <textarea name="message" id="message" className="form-control" rows="9" cols="25"
                                                          placeholder="Пожелания" value={this.state.message} onChange={(e)=> this.handleMessageChange(e)}></textarea>
                                            </div>
                                        </div>
                                        <div className="col-md-12">
                                            <button type="submit" className="btn btn-success pull-right" id="btnContactUs"  onClick={(e)=> this.btnSendInfo(e) }>
                                                Заказать</button>
                                        </div>
                                    </div>
                                </form>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
        )
    }
}



const mapDispatchToProps = {
    startAddOrder
}

export default connect(null, mapDispatchToProps)(FromDelivery)
