// containers/basket/yandex.js
import React, { Component } from 'react';
import { YMaps, Map, Circle, GeoObject } from 'react-yandex-maps';



export class YandexDelivery extends Component {
  render() {
    const mapState = { center: [55.155937, 61.410854], zoom: 11 };
    const {userCoordinats} = this.props

    console.log('userCoordinats = ' +  JSON.parse('[' + userCoordinats + ']'))

    return (<YMaps>
      <Map state={mapState} width='100%' height='600px'>
        <GeoObject
          geometry={{
            type: 'Point',
            coordinates: [55.155937, 61.410854]
          }}
          properties={{
            iconContent: 'Доставка 50 рублей'
          }}
          options={{
            preset: 'islands#darkGreenStretchyIcon'
          }}
        />
        <GeoObject
          // 150 рублей
          geometry={{
            type: 'Point',
            coordinates: [55.194789, 61.455887]
          }}
          properties={{
            iconContent: 'Доставка 150 рублей'
          }}
          options={{
            preset: 'islands#redStretchyIcon'
          }}
        />
        <GeoObject
          // точка клиента
          geometry={{
            type: 'Point',
            coordinates:  JSON.parse('[' + userCoordinats + ']'),
          }} 
          options={{
            preset: 'twirl#blackDotIcon',
            iconColor: '#000000'
          }}
        />
        <Circle
          geometry={{
            coordinates: [55.155937, 61.410854],
            radius: 4000
          }}
          options={{
            draggable: false,
            fillColor: '#ffff0022',
            strokeColor: '#3caa3c88',
            strokeOpacity: 0.8,
            strokeWidth: 5,
          }}
        />
        <Circle
          geometry={{
            coordinates: [55.155937, 61.410854],
            radius: 9000
          }}
          options={{
            draggable: false,
            fillColor: 'rgba(237, 235, 235, 0.3)',
            strokeColor: '#cf0000',
            strokeOpacity: 0.8,
            strokeWidth: 5,
          }}
        />
      </Map>
    </YMaps>
    )
  }
}
