// utils/common.js
module.exports = {
    createArrayTypeOfGoods : (data) => {
        if(data == null) return []
        let allData = []
        let keys = Object.keys(data);
        let values = Object.values(data);
        for (let i = 0; i < Object.keys(data).length; i++) {
            let pizza = {
                key: keys[i],
                id: values[i].id,
                categoryId: values[i].categoryId,
                name: values[i].name,
                description: values[i].description,
                price: values[i].price,
                image: values[i].image
            }
            allData.push(pizza)
        }
        return allData
    },
    getNameCategoryById : (id) => {
        switch(id){
            case 1:
            return 'Пицца'
            case 2:
            return 'Домашняя паста'
            case 3:
            return 'Салаты'
            case 4:
            return 'Десерты'
            case 5:
            return 'Перекусы'
            case 6:
            return 'Горячие напитки'
            case 7:
            return 'Холодные напитки'
        }
    }
}
