// containers/category/index.js
import React from 'react';
import {GridList, GridTile} from 'material-ui/GridList';
import IconButton from 'material-ui/IconButton';
import { browserHistory } from 'react-router'
const styles = {
    root: {
      display: 'flex',
      flexWrap: 'wrap',
      justifyContent: 'space-around',
      marginTop: 68
    },
    gridList: {
      width: '45%',
      overflowY: 'auto',
    },
  };

const tilesData = [
  {
    img: '/uploads/category/pizza.jpg',
    title: 'Пицца',
    url:'pizza'
  },
  {
    img: '/uploads/category/home_pasta.jpg',
    title: 'Домашняя паста',
    url:'home_pasta'
  },
  {
    img: '/uploads/category/snikers.jpg',
    title: 'Перекусы',
    url:'snikers'
  },
  {
    img: '/uploads/category/salats.jpg',
    title: 'Салаты',
    url:'salats'
  },
  {
    img: '/uploads/category/deserts.jpg',
    title: 'Десерты',
    url:'deserts'
  },
  {
    img: '/uploads/category/hot_drink.jpg',
    title: 'Горячие напитки',
    url:'hot_drink'
  },
  {
    img: '/uploads/category/cold_drinks.png',
    title: 'Холодные напитки',
    url:'cold_drinks'
  }
];

/**
 * This example demonstrates the horizontal scrollable single-line grid list of images.
 */
const Category = () => (
    <div style={styles.root}>
    <GridList
      cellHeight={180}
      style={styles.gridList} cols={window.screen.availWidth > 1000 ? 4 : 1}
    >
      {tilesData.map((tile) => (
        <GridTile
          key={tile.img}
          title={tile.title}
          titleBackground = 'rgba(45, 177, 4, 0.7)'
          actionIcon={<IconButton onClick={() => browserHistory.push(tile.url)}><svg fill="white" height="24" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg">
          <path d="M0 0h24v24H0zm18.31 6l-2.76 5z" fill="none" />
          <path d="M11 9h2V6h3V4h-3V1h-2v3H8v2h3v3zm-4 9c-1.1 0-1.99.9-1.99 2S5.9 22 7 22s2-.9 2-2-.9-2-2-2zm10 0c-1.1 0-1.99.9-1.99 2s.89 2 1.99 2 2-.9 2-2-.9-2-2-2zm-9.83-3.25l.03-.12.9-1.63h7.45c.75 0 1.41-.41 1.75-1.03l3.86-7.01L19.42 4h-.01l-1.1 2-2.76 5H8.53l-.13-.27L6.16 6l-.95-2-.94-2H1v2h2l3.6 7.59-1.35 2.45c-.16.28-.25.61-.25.96 0 1.1.9 2 2 2h12v-2H7.42c-.13 0-.25-.11-.25-.25z" />
      </svg></IconButton>}
      titlePosition='bottom'
          titleStyle={{ fontSize: 12 }}
        >
          <img src={tile.img} />
        </GridTile>
      ))}
    </GridList>
  </div>
);

export default Category;
