// api/mockPizza.js
export default [
    {
        id: "1",
        categoryId: "1",
        name: "Романа",
        description: "Томатный соус, моццарелла, ветчина и грибы. 480Гр.",
        price: 380,
        image: "/uploads/romana.jpg"
    },
    {
        id: "2",
        categoryId: "1",
        name: "Маргаритона",
        description: "Томатный соус, моццарелла кусочками, базилик, и целый шарик моццареллы. 450Гр.",
        price: 390,
        image: "/uploads/margaritona.jpg",
    },
    {
        id: "3",
        categoryId: "4",
        name: "Маргарита",
        description: "Томатный соус, моццарелла и базилик. 350Гр.",
        price: 280,
        image: "/uploads/margarita.jpg",
    },
    {
        id: "4",
        categoryId: "5",
        name: "Примавера",
        description: "Томатный соус, моццарелла, сыровяленая ветчина, руккола и помидоры черри. 470Гр.",
        price: 430,
        image: "/uploads/primavera.jpg",
    },
    {
        id: "5",
        categoryId: "3",
        name: "Четыре сыра",
        description: "Томатный соус, моццарелла, сыр \"Гауда\", \"Горгонзола\" и солёная рикотта. 430Гр.",
        price: 340,
        image: "/uploads/fourcheese.jpg",
    },
    {
        id: "6",
        categoryId: "2",
        name: "Диавола",
        description: "Томатный соус, моццарелла, салями и свежий перец чили. 440Гр.",
        price: 410,
        image: "/uploads/evil.jpg",
    }
    ,
    {
        id: "7",
        categoryId: "2",
        name: "Галлетто",
        description: "Томатный соус, моццарелла, сладкий перец и кусочки курицы, \n" +
        "обжаренные с бальзамическим уксусом. 440Гр.",
        price: 390,
        image: "/uploads/galetto.jpg",
    },
    {
        id: "8",
        categoryId: "2",
        name: "Шеф",
        description: "Томатный соус, зелёный лук, рикотта, анчоусы, оригано и оливковое масло. 330Гр.",
        price: 390,
        image: "/uploads/shef.jpg",
    },
    {
        id: "9",
        categoryId: "2",
        name: "Липен",
        description: " Томатный соус, моццарелла, рикотта, колбаса, бекон, ветчина и помидоры дольками. 540Гр.",
        price: 480,
        image: "/uploads/lipen.jpg",
    },
    {
        id: "10",
        categoryId: "3",
        name: "Спагетти с томатным соусом и базиликом ",
        description: " Спагетти с томатным соусом и базиликом .  210Гр.",
        price: 270,
        image: "/uploads/spagetti.jpg",
    },
    {
        id: "11",
        categoryId: "3",
        name: "Спагетти с соусом \"Песто\"",
        description: "Соус из базилика с моццареллой и помидорами-черри. 220Гр.",
        price: 370,
        image: "/uploads/pesto.jpg",
    },
    {
        id: "12",
        categoryId: "3",
        name: "Спагетти \"Карбонара\"",
        description: "Спагетти с беконом, сливками и яйцом. 260Гр.",
        price: 350,
        image: "/uploads/karbonara.jpg",
    },
    {
        id: "13",
        categoryId: "3",
        name: "Тальятелле с рагу \"Болоньезе\"",
        description: "Тальятелле с рагу из мясного фарша с томатным соусом. 280Гр.",
        price: 370,
        image: "/uploads/boloneze.jpg",
    },
    {
        id: "14",
        categoryId: "3",
        name: "Тальятелле с креветками",
        description: "Тальятелле с креветками, спаржей, помидорами-черри со сливками. 280Гр.",
        price: 410,
        image: "/uploads/talliatti.jpg",
    },
    {
        id: "15",
        categoryId: "3",
        name: "Диавола",
        description: "С мясным рагу. 300Гр.",
        price: 380,
        image: "/uploads/lozania.jpg",
    },
    {
        id: "16",
        categoryId: "3",
        name: "Паста \"Ласточкины гнёзда\"",
        description: "Ветчина, моццарелла и соус \"Бешамель\". 2500Гр.",
        price: 350,
        image: "/uploads/evil.jpg",
    },
    {
        id: "17",
        categoryId: "2",
        name: "Диавола",
        description: "Томатный соус, моццарелла, салями и свежий перец чили. 440Гр.",
        price: 410,
        image: "/uploads/evil.jpg",
    },
    {
        id: "18",
        categoryId: "2",
        name: "Диавола",
        description: "Томатный соус, моццарелла, салями и свежий перец чили. 440Гр.",
        price: 410,
        image: "/uploads/evil.jpg",
    },
    {
        id: "19",
        categoryId: "2",
        name: "Диавола",
        description: "Томатный соус, моццарелла, салями и свежий перец чили. 440Гр.",
        price: 410,
        image: "/uploads/evil.jpg",
    },
    {
        id: "20",
        categoryId: "2",
        name: "Диавола",
        description: "Томатный соус, моццарелла, салями и свежий перец чили. 440Гр.",
        price: 410,
        image: "/uploads/evil.jpg",
    },
    {
        id: "21",
        categoryId: "2",
        name: "Диавола",
        description: "Томатный соус, моццарелла, салями и свежий перец чили. 440Гр.",
        price: 410,
        image: "/uploads/evil.jpg",
    },
    {
        id: "22",
        categoryId: "2",
        name: "Диавола",
        description: "Томатный соус, моццарелла, салями и свежий перец чили. 440Гр.",
        price: 410,
        image: "/uploads/evil.jpg",
    },
    {
        id: "23",
        categoryId: "2",
        name: "Диавола",
        description: "Томатный соус, моццарелла, салями и свежий перец чили. 440Гр.",
        price: 410,
        image: "/uploads/evil.jpg",
    },
    {
        id: "24",
        categoryId: "2",
        name: "Диавола",
        description: "Томатный соус, моццарелла, салями и свежий перец чили. 440Гр.",
        price: 410,
        image: "/uploads/evil.jpg",
    },
    {
        id: "25",
        categoryId: "2",
        name: "Диавола",
        description: "Томатный соус, моццарелла, салями и свежий перец чили. 440Гр.",
        price: 410,
        image: "/uploads/evil.jpg",
    },
    {
        id: "26",
        categoryId: "2",
        name: "Диавола",
        description: "Томатный соус, моццарелла, салями и свежий перец чили. 440Гр.",
        price: 410,
        image: "/uploads/evil.jpg",
    }
]
