// api/mockDataTmp.js
var pizzess = [
    {
        id: "1",
        categoryId: "1",
        name: "Романа",
        description: "Томатный соус, моццарелла, ветчина и грибы. 480Гр.",
        price: 380,
        image: "/uploads/romana.jpg"
    },
    {
        id: "2",
        categoryId: "1",
        name: "Маргаритона",
        description: "Томатный соус, моццарелла кусочками, базилик, и целый шарик моццареллы. 450Гр.",
        price: 390,
        image: "/uploads/margaritona.jpg",
    },
    {
        id: "3",
        categoryId: "4",
        name: "Маргарита",
        description: "Томатный соус, моццарелла и базилик. 350Гр.",
        price: 280,
        image: "/uploads/margarita.jpg",
    },
    {
        id: "4",
        categoryId: "5",
        name: "Примавера",
        description: "Томатный соус, моццарелла, сыровяленая ветчина, руккола и помидоры черри. 470Гр.",
        price: 430,
        image: "/uploads/primavera.jpg",
    },
    {
        id: "5",
        categoryId: "3",
        name: "Четыре сыра",
        description: "Томатный соус, моццарелла, сыр Гауда, Горгонзола и солёная рикотта. 430Гр.",
        price: 340,
        image: "/uploads/fourcheese.jpg",
    },
    {
        id: "6",
        categoryId: "2",
        name: "Диавола",
        description: "Томатный соус, моццарелла, салями и свежий перец чили. 440Гр.",
        price: 410,
        image: "/uploads/evil.jpg",
    }
    ,
    {
        id: "7",
        categoryId: "2",
        name: "Галлетто",
        description: "Томатный соус, моццарелла, сладкий перец и кусочки курицы обжаренные с бальзамическим уксусом. 440Гр.",
        price: 390,
        image: "/uploads/galetto.jpg",
    },
    {
        id: "8",
        categoryId: "2",
        name: "Шеф",
        description: "Томатный соус, зелёный лук, рикотта, анчоусы, оригано и оливковое масло. 330Гр.",
        price: 390,
        image: "/uploads/shef.jpg",
    },
    {
        id: "9",
        categoryId: "2",
        name: "Липен",
        description: "Томатный соус, моццарелла, рикотта, колбаса, бекон, ветчина и помидоры дольками. 540Гр.",
        price: 480,
        image: "/uploads/lipen.jpg",
    },
    {
        id: "10",
        categoryId: "3",
        name: "Спагетти с томатным соусом и базиликом",
        description: "Спагетти с томатным соусом и базиликом. 210Гр.",
        price: 270,
        image: "/uploads/spagetti.jpg",
    },
    {
        id: "11",
        categoryId: "3",
        name: "Спагетти с соусом Песто",
        description: "Соус из базилика с моццареллой и помидорами-черри. 220Гр.",
        price: 370,
        image: "/uploads/pesto.jpg",
    },
    {
        id: "12",
        categoryId: "3",
        name: "Спагетти Карбонара",
        description: "Спагетти с беконом, сливками и яйцом. 260Гр.",
        price: 350,
        image: "/uploads/karbonara.jpg",
    },
    {
        id: "13",
        categoryId: "3",
        name: "Тальятелле с рагу Болоньезе",
        description: "Тальятелле с рагу из мясного фарша с томатным соусом. 280Гр.",
        price: 370,
        image: "/uploads/boloneze.jpg",
    },
    {
        id: "14",
        categoryId: "3",
        name: "Тальятелле с креветками",
        description: "Тальятелле с креветками, спаржей, помидорами-черри со сливками. 280Гр.",
        price: 410,
        image: "/uploads/talliatti.jpg",
    },
    {
        id: "15",
        categoryId: "3",
        name: "Лазанья Болоньезе",
        description: "С мясным рагу. 300Гр.",
        price: 380,
        image: "/uploads/lozania.jpg",
    },
    {
        id: "16",
        categoryId: "3",
        name: "Паста Ласточкины гнёзда",
        description: "Ветчина, моццарелла и соус Бешамель. 250Гр.",
        price: 350,
        image: "/uploads/evil.jpg",
    },
    {
        id: "17",
        categoryId: "4",
        name: "Сицилийские Аранчини",
        description: "Мясное рагу, рис, панировка. 180Гр.",
        price: 110,
        image: "/uploads/aranchi.jpg",
    },
    {
        id: "18",
        categoryId: "4",
        name: "Пьядина Романьола",
        description: " Лепёшка с сыровяленой ветчиной, сыром Страччателла и рукколой. 210Гр.",
        price: 270,
        image: "/uploads/padina.jpg",
    },
    {
        id: "19",
        categoryId: "4",
        name: "Фритто-ди-маре",
        description: "Креветки, кальмары и другие морепродукты.250Гр.",
        price: 250,
        image: "/uploads/fritto.jpg",
    },
    {
        id: "20",
        categoryId: "4",
        name: "Трис из брускетт ",
        description: " Помидоры, базилик и чеснок - Сало, мёд и розмарин Вяленые помидоры, моццарелла и анчоусы. 90Гр.",
        price: 150,
        image: "/uploads/tris.jpg",
    },
    {
        id: "21",
        categoryId: "5",
        name: "Пицца в ассортименте на противне",
        description: "Кусочками. С колбасой и беконом или болгарским перцем и грибами. 130Гр.",
        price: 105,
        image: "/uploads/pizzapto.jpeg",
    },
    {
        id: "22",
        categoryId: "5",
        name: "Средиземноморский салат",
        description: "Руккола, помидоры черри, яблоко, авокадо и моццарелла. 190Гр.",
        price: 280,
        image: "/uploads/salat.jpg",
    },
    {
        id: "23",
        categoryId: "5",
        name: "Буррата",
        description: "Буррата, помидоры черри и листья зелёного салата. 280Гр.",
        price: 380,
        image: "/uploads/buratta.jpg",
    },
    {
        id: "24",
        categoryId: "5",
        name: "Панцанелла от шефа",
        description: "Томаты, огурцы, красный лук, анчоусы, кусочки хлеба. 2300Гр.",
        price: 310,
        image: "/uploads/panazea.jpg",
    },
    {
        id: "25",
        categoryId: "5",
        name: "Сицилия",
        description: " Руккола, огурцы, креветки, помидорки и фрукты (апельсины или земляника). 260Гр.",
        price: 350,
        image: "/uploads/sicilia.jpg",
    },
    {
        id: "31",
        categoryId: "5",
        name: "Буон густо",
        description: "Кусочки курицы, манго, авокадо, листовой салат и кешью. 230Гр.",
        price: 390,
        image: "/uploads/gusto.jpg",
    },
    {
        id: "26",
        categoryId: "5",
        name: "Капрезе ",
        description: "Моццарелла, помидоры-черри и базилик. 220Гр.",
        price: 330,
        image: "/uploads/caprese.jpg",
    } ,
    {
        id: "27",
        categoryId: "6",
        name: "Сицилийские канноли",
        description: "С кремом из рикотты . 100Гр.",
        price: 140,
        image: "/uploads/cannolii.jpg",
    },
    {
        id: "28",
        categoryId: "6",
        name: "Панна Котта",
        description: "Сливки, сахар и соус из сезонных фруктов. 120Гр.",
        price: 200,
        image: "/uploads/panna.jpg",
    }
    ,
    {
        id: "29",
        categoryId: "6",
        name: "Тирамису",
        description: "С кремом из Маскарпоне и печеньем Савойарди\ 130Гр.",
        price: 250,
        image: "/uploads/tiramisu.jpg",
    },
    {
        id: "30",
        categoryId: "6",
        name: "Семифреддо с сезонными фруктами",
        description: "Мягкое мороженое. 100Гр.",
        price: 190,
        image: "/uploads/semifreddo.jpg",
    }
]

exports.MockPizzes = pizzess;
