// api/mockPizza.js
export default [
    {
        id: "1",
        categoryId: "1",
        name: "Романа",
        description: "Томатный соус, моццарелла, ветчина и грибы. 480Гр.",
        price: 380,
        image: "/uploads/romana.jpg",
        cpu: "1.3GHz Apple A6",
        camera: "8mp (3264x2448)",
        size: "124.4mm x 59.2mm x 8.97mm (4.9 x 2.33 x 0.35)",
        weight: "132 grams (4.7 ounces) with battery",
        display: "4.0 326 pixel density",
        battery: "1480 mAh",
        memory: "16GB, 32GB and RAM 1 GB"
    },
    {
        id: "2",
        categoryId: "1",
        name: "Маргаритона",
        description: "Томатный соус, моццарелла кусочками, базилик, и целый шарик моццареллы. 450Гр.",
        price: 390,
        image: "/uploads/margaritona.jpg",
        cpu: "1.3GHz Apple A6",
        camera: "8mp (3264x2448)",
        size: "124.4mm x 59.2mm x 8.97mm (4.9 x 2.33 x 0.35)",
        weight: "132 grams (4.7 ounces) with battery",
        display: "4.0 326 pixel density",
        battery: "1480 mAh",
        memory: "16GB, 32GB and RAM 1 GB"
    },
    {
        id: "3",
        categoryId: "4",
        name: "Маргарита",
        description: "Томатный соус, моццарелла и базилик. 350Гр.",
        price: 280,
        image: "/uploads/margarita.jpg",
        cpu: "1.3GHz Apple A6",
        camera: "8mp (3264x2448)",
        size: "124.4mm x 59.2mm x 8.97mm (4.9 x 2.33 x 0.35)",
        weight: "132 grams (4.7 ounces) with battery",
        display: "4.0 326 pixel density",
        battery: "1480 mAh",
        memory: "16GB, 32GB and RAM 1 GB"
    },
    {
        id: "4",
        categoryId: "5",
        name: "Примавера",
        description: "Томатный соус, моццарелла, сыровяленая ветчина, руккола и помидоры черри. 470Гр.",
        price: 430,
        image: "/uploads/primavera.jpg",
        cpu: "1.3GHz Apple A6",
        camera: "8mp (3264x2448)",
        size: "124.4mm x 59.2mm x 8.97mm (4.9 x 2.33 x 0.35)",
        weight: "132 grams (4.7 ounces) with battery",
        display: "4.0 326 pixel density",
        battery: "1480 mAh",
        memory: "16GB, 32GB and RAM 1 GB"
    },
    {
        id: "5",
        categoryId: "3",
        name: "Четыре сыра",
        description: "Томатный соус, моццарелла, сыр \"Гауда\", \"Горгонзола\" и солёная рикотта. 430Гр.",
        price: 340,
        image: "/uploads/fourcheese.jpg",
        cpu: "1.3GHz Apple A6",
        camera: "8mp (3264x2448)",
        size: "124.4mm x 59.2mm x 8.97mm (4.9 x 2.33 x 0.35)",
        weight: "132 grams (4.7 ounces) with battery",
        display: "4.0 326 pixel density",
        battery: "1480 mAh",
        memory: "16GB, 32GB and RAM 1 GB"
    },
    {
        id: "6",
        categoryId: "2",
        name: "Диавола",
        description: "Томатный соус, моццарелла, салями и свежий перец чили. 440Гр.",
        price: 410,
        image: "/uploads/evil.jpg",
        cpu: "1.3GHz Apple A6",
        camera: "8mp (3264x2448)",
        size: "124.4mm x 59.2mm x 8.97mm (4.9 x 2.33 x 0.35)",
        weight: "132 grams (4.7 ounces) with battery",
        display: "4.0 326 pixel density",
        battery: "1480 mAh",
        memory: "16GB, 32GB and RAM 1 GB"
    }
]
