// api/mockPizza.js
export default [
    {
        id: "1",
        categoryId: "1",
        name: "Романа",
        description: "Томатный соус, моццарелла, ветчина и грибы. 480Гр.",
        price: 380,
        image: "/uploads/romana.jpg"
    },
    {
        id: "2",
        categoryId: "1",
        name: "Маргаритона",
        description: "Томатный соус, моццарелла кусочками, базилик, и целый шарик моццареллы. 450Гр.",
        price: 390,
        image: "/uploads/margaritona.jpg",
    },
    {
        id: "3",
        categoryId: "4",
        name: "Маргарита",
        description: "Томатный соус, моццарелла и базилик. 350Гр.",
        price: 280,
        image: "/uploads/margarita.jpg",
    },
    {
        id: "4",
        categoryId: "5",
        name: "Примавера",
        description: "Томатный соус, моццарелла, сыровяленая ветчина, руккола и помидоры черри. 470Гр.",
        price: 430,
        image: "/uploads/primavera.jpg",
    },
    {
        id: "5",
        categoryId: "3",
        name: "Четыре сыра",
        description: "Томатный соус, моццарелла, сыр \"Гауда\", \"Горгонзола\" и солёная рикотта. 430Гр.",
        price: 340,
        image: "/uploads/fourcheese.jpg",
    },
    {
        id: "6",
        categoryId: "2",
        name: "Диавола",
        description: "Томатный соус, моццарелла, салями и свежий перец чили. 440Гр.",
        price: 410,
        image: "/uploads/evil.jpg",
    }
]
